jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -310,6 +310,9 @@
           status-add-test-results: false
           trigger-phrase: 'verify|test'
           cmd: |
+            # verify-godeps compares against upstream, but remote/master might be stale
+            # if .git was retained between runs. Update it explicitly here.
+            git fetch remote master:refs/remotes/remote/master
```

Comment:
ok, now I'm just confused. the Jenkins log shows
```
/usr/bin/git -c core.askpass=true fetch --tags --progress https://github.com/kubernetes/kubernetes +refs/heads/*:refs/remotes/upstream/* # timeout=20
```
so I don't understand how this should be stale.